graphql_compiler/cost_estimation/.#interval.py

```diff
@@ -0,0 +1 @@
+bojan@kadmin-XPS-15-9570.9333:1577977235
```

Comment:
Consider updating the `.gitignore` or your local exclude file to avoid checking in files like this